ice40/utils/fasm_icebox/ice40_feature.py

```diff
@@ -0,0 +1,80 @@
+import re
+from collections import namedtuple
+
+FasmEntry = namedtuple('FasmDbEntry', 'feature bits')
+IceDbEntry = namedtuple('IceDbEntry', 'tile_type loc bits names idx')
+
+class Feature(object):
+
+    def __init__(self, tile_type, loc, bit_tuples, parts, idx=None):
+        self.tile_type = tile_type
+        self.loc = loc
+        self.bit_tuples = bit_tuples
+        self.parts = parts
+        self.idx = idx
+
+    @staticmethod
+    def _parse_icedb_bit(bit):
```

Comment:
Why static method versus a free function?

Reply:
I was scoping it to as a helper to the class, but moved it out if someone wants to use it elsewhere.